docs/core/diagnostics/distributed-tracing-collection-walkthroughs.md

```diff
@@ -0,0 +1,326 @@
+            {
+                await Task.Delay(500);
+            }
+        }
+
+        static async Task StepTwo()
+        {
+            using (Activity a = s_source.StartActivity("StepTwo"))
+            {
+                await Task.Delay(1000);
+            }
+        }
+    }
+}
+```
+
+Running the app does not log anything yet
+```dotnetcli
+> dotnet run
+Example work done
+```
+
+### Add logging using OpenTelemetry
+
+Add the [OpenTelemetry](https://www.nuget.org/packages/OpenTelemetry/) and 
+[OpenTelemetry.Exporter.Console](https://www.nuget.org/packages/OpenTelemetry.Exporter.Console/) NuGet packages.
+
+```dotnetcli
+dotnet add package OpenTelemetry
+dotnet add package OpenTelemetry.Exporter.Console
+```
+
+Update Program.cs with additional using statments:
+```C#
+using OpenTelemetry;
+using OpenTelemetry.Trace;
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+```
+
+And update Main() to create the OpenTelemetry TracerProvider:
+```C#
+        public static async Task Main()
+        {
+            using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                .SetSampler(new AlwaysOnSampler())
+                // Add more libraries
+                .AddSource("Sample.DistributedTracing")
+                // Add more exporters
+                .AddConsoleExporter()
+                .Build();
+
+            await DoSomeWork();
+            Console.WriteLine("Example work done");
+        }
+```
+
+Now the app logs distributed trace information to the console:
+```dotnetcli
+> dotnet run
+Activity.Id:          00-35c0e68b0dac3c49be08a9d9cab32579-0b7477e11aa20d40-01
+Activity.ParentId:    00-35c0e68b0dac3c49be08a9d9cab32579-d95f666d24193f40-01
+Activity.DisplayName: StepOne
+Activity.Kind:        Internal
+Activity.StartTime:   2021-03-15T01:58:10.7661575Z
+Activity.Duration:    00:00:00.5013965
+Resource associated with Activity:
+    service.name: unknown_service:temp
+
+Activity.Id:          00-35c0e68b0dac3c49be08a9d9cab32579-7ba0fc6d480c9841-01
+Activity.ParentId:    00-35c0e68b0dac3c49be08a9d9cab32579-d95f666d24193f40-01
+Activity.DisplayName: StepTwo
+Activity.Kind:        Internal
+Activity.StartTime:   2021-03-15T01:58:11.2894853Z
+Activity.Duration:    00:00:01.0188689
+Resource associated with Activity:
+    service.name: unknown_service:temp
+
+Activity.Id:          00-35c0e68b0dac3c49be08a9d9cab32579-d95f666d24193f40-01
+Activity.DisplayName: SomeWork
+Activity.Kind:        Internal
+Activity.StartTime:   2021-03-15T01:58:10.7647839Z
+Activity.Duration:    00:00:01.5450288
+Resource associated with Activity:
+    service.name: unknown_service:temp
+
+Example work done
+```
+
+The console exporter is helpful for quick examples or local development but in a production deployment
+you will probably want to send logs to a central logging store. OpenTelemetry supports a variety
+of different logging destinations using different
```

Comment:
logging - I'll try to avoid this term :)

Reply:
Reworded : )